x/concentrated-liquidity/spread_rewards_test.go

```diff
@@ -1359,6 +1360,27 @@ func (s *KeeperTestSuite) TestUpdatePosValueToInitValuePlusGrowthOutside() {
 	}
 }
 
+func NewRandomizedPositions() Positions {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Let's make the seed deterministic please

Reply:
but if we use a deterministic seed, won't it mean, that for each test, which gets a positions struct with this function, the initialized object will be the same? afaiu, there will be no point in using randomness at all

Reply:
Can we find a way to use a different but deterministic and known seed for every use of `NewRandomizedPositions`?

Having a random seed based on time that developers don't know isn't very useful because even if you get a test failure, you can't reproduce it